src/System.Text.Json/tests/Serialization/ExtensionDataTests.cs

```diff
@@ -262,5 +340,16 @@ public class ClassWithReference
 
             public ClassWithExtensionProperty MyReference { get; set; }
         }
+
+        // Match model in issue https://github.com/dotnet/corefx/issues/38238
+        public class TestModel_Issue_38238
```

Comment:
@ahsonkhan added reference to issue...I don't know if it's ok a strong ref.
Let me know.

Reply:
This is redundant from the other tests. Not required IMO.

Reply:
cc: @ahsonkhan also to me it's redundant, let me know if I need to remove.

Reply:
I generally have a habit of making sure the initially reported test case/scenario is covered by a unit test. It may be redundant, sure, but I don't see the harm here of adding a test like this. However, I don't feel strongly about this and would be fine with either.

Reply:
I'll remove reduntant.